samza-shell/src/main/bash/run-app.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
run-app.sh will replace run-job.sh going forward. But for legacy jobs, we can either keep run-job.sh or force them to upgrade run script. I would prefer keeping it for now so it won't cause panic for our users :)